mod/wikirate/lib/open_corporates/api.rb

```diff
@@ -34,11 +34,6 @@ def fetch_jurisdictions
       # @return the full json response converted to a hash
       def fetch *query_args
         JSON.parse json_response(*query_args)
```

Comment:
what happens to parse errors?  (Looks like it may have been incorrectly rescuing those before...)